catalystwan/workflows/tenant_migration.py

```diff
@@ -127,15 +127,9 @@ def migration_preconditions_check(
     logger.info("Checking if migrated devices can reach target validator...")
     conn_check = False
     if origin_session.session_type == SessionType.PROVIDER:
-        with create_manager_session(
-            url=origin_session.url,
-            username=origin_session.username,
-            password=origin_session.password,
-            port=origin_session.port,
-            subdomain=tenant.subdomain,
-            logger=origin_session.logger,
-        ) as provider_as_tenant_session:
-            conn_check = check_control_connectivity_from_edge_devices(provider_as_tenant_session, validator)
+        origin_session.subdomain = tenant.subdomain
```

Comment:
session is not closed as in original, maybe use:
```suggestion
        origin_session.subdomain = tenant.subdomain
        with origin_session.login() as provider_as_tenant_session:
            conn_check = check_control_connectivity_from_edge_devices(provider_as_tenant_session, validator)
```

Reply:
Changed as suggested